### PR TITLE
feat: `ddk configurator` build

### PR DESCRIPTION
**Pull request by malachi-constant**
### Detail
- [ ] #246
- [ ] #247  
- [ ] #248 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/config/configurator.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  public visit(node: constructs.IConstruct): void {
+    if (cdk.CfnResource.isCfnResource(node) && node.cfnResourceType == this.resourceType) {
+      node.addPropertyOverride(this.propertyName, this.propertyValue);
+    }
+  }
+}
+
+export class Configurator {
+  private readonly config: any;
+  constructor(scope: constructs.Construct, configData: any, environmentId?: string) {
```
Comment (malachi-constant) [inline src/config/configurator.ts:23]: Quite a bird’s nest here, maybe there’s a way to clean this up

File: src/config/configurator.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  public visit(node: constructs.IConstruct): void {
+    if (cdk.CfnResource.isCfnResource(node) && node.cfnResourceType == this.resourceType) {
+      node.addPropertyOverride(this.propertyName, this.propertyValue);
+    }
+  }
+}
+
+export class Configurator {
+  private readonly config: any;
+  constructor(scope: constructs.Construct, config: string | object, environmentId?: string) {
```
Comment (jaidisido) [inline src/config/configurator.ts:24]: Is the idea to different config strategies (e.g. JSON, Yaml...) in a future PR?

File: test/test-config.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "environments": {
+    "dev": {
+      "account": "222222222222",
+      "region": "us-east-1",
+      "resources": {
+        "AWS::Lambda::Function": {
```
Comment (jaidisido) [inline test/test-config.json:7]: So the current identifier used is the resource type? Meaning all Lambda functions would be impacted by the Aspect? Wouldn't that be too broad?

Comment (malachi-constant) [reply test/test-config.json:7]: Yea, we should be able to implement by construct `id` as well. I can add that support in a future PR.

File: src/config/configurator.ts (hunk 2 above)
Comment (malachi-constant) [reply src/config/configurator.ts:24]: Correct

Comment (malachi-constant) [reply src/config/configurator.ts:24]: Actually, will update to support YAML in this PR.

File: test/test-config.json (hunk 3 above)
Comment (malachi-constant) [reply test/test-config.json:7]: Will add in this PR.

**Comment by malachi-constant** [comment]
Closed in favor of #254

**Pull request closed without merging**